app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -473,15 +473,64 @@ def center_toolbar_filename_classic
     else
       # show_list and show screens
       unless @in_a_form
-        if %w(auth_key_pair_cloud availability_zone host_aggregate cloud_object_store_object cloud_object_store_container cloud_tenant
-              cloud_volume cloud_volume_backup cloud_volume_snapshot configuration_job container_group container_node container_service
-              ems_cloud ems_cluster ems_container ems_datawarehouse ems_middleware container_project container_route container_replicator container_image
-              ems_network security_group floating_ip cloud_subnet network_router network_topology network_port cloud_network load_balancer
-              container_image_registry ems_infra flavor host container_build infra_networking infra_topology ems_storage
-              container_topology middleware_topology cloud_topology middleware_server
-              middleware_deployment middleware_datasource middleware_domain middleware_server_group middleware_messaging
-              orchestration_stack resource_pool storage_manager container_template
-              ems_block_storage ems_object_storage timeline usage).include?(@layout)
+        if %w(auth_key_pair_cloud
+              availability_zone
+              host_aggregate
+              cloud_object_store_object
+              cloud_object_store_container
+              cloud_tenant
+              cloud_volume
+              cloud_volume_backup
+              cloud_volume_snapshot
+              configuration_job
+              container_group
+              container_node
+              container_service
+              ems_cloud
+              ems_cluster
+              ems_container
+              ems_datawarehouse
+              ems_middleware
+              container_project
+              container_route
+              container_replicator
+              container_image
+              ems_network
+              security_group
+              floating_ip
+              cloud_subnet
+              network_router
+              network_topology
+              network_port
+              cloud_network
+              load_balancer
+              container_image_registry
+              ems_infra
+              ems_physical_infra
+              flavor
+              host
+              container_build
+              infra_networking
+              infra_topology
+              ems_storage
+              container_topology
+              middleware_topology
+              cloud_topology
+              middleware_server
+              middleware_deployment
+              middleware_datasource
+              middleware_domain
+              middleware_server_group
+              middleware_messaging
+              orchestration_stack
+              physical_infra_topology
+              resource_pool
+              storage_manager
+              container_template
+              ems_block_storage
+              ems_object_storage
+              timeline
+              usage).include?(@layout)
```

Comment:
@dclarizio @martinpovolny 2 requests here for future PRs:
- Please alphabetize this list
- Please move the list to a well-named constant so code is more readable